httpie/output/formatters/json.py

```diff
@@ -17,6 +17,10 @@ def format_body(self, body, mime):
         ]
         if (self.kwargs['explicit_json'] or
                 any(token in mime for token in maybe_json)):
+            if 'xssi_prefix' in self.kwargs:
+                token = self.kwargs["xssi_prefix"]
+                if body[0:len(token)] == token:
```

Comment:
Would it not be simpler to do:

```py
if body.startswith(token):
```

It looks as though `token` being `''` won't get through to here because otherwise you'll always find this condition to be true and you'll always make a copy.

That said, even if you specify the common value that @jkbrzt found, this should be a relatively cheap check (to use startswith).